Source/com/drew/metadata/mp4/Mp4HandlerFactory.java

```diff
@@ -31,7 +31,8 @@
     private static final String HANDLER_VIDEO_MEDIA             = "vide";
     private static final String HANDLER_HINT_MEDIA              = "hint";
     private static final String HANDLER_TEXT_MEDIA              = "text";
-    private static final String HANDLER_META_MEDIA              = "meta";
+    private static final String HANDLER_NRT_META_MEDIA          = "meta";
+    private static final String HANDLER_META_MEDIA              = "mdir";
```

Comment:
The "ilst" box lies under the "mdir" (metadata) handler.

Documented under "QuickTime Handler Keys" https://exiftool.org/TagNames/QuickTime.html